### PR TITLE
[CFM-1254] Fixes feedback about the highlight button

### PR DESCRIPTION
**Pull request by DavidHernandez**
Issue: https://issuetracker.amplexor.com/jira/browse/CFM-1254

- Replaces promote icon with a star.
- Improves the theming of the promote icon.
- Adds the promote icon to all the search result view modes.

### Group dashboard - desktop
pinned
### Tablet
pinned-tablet
### Mobile
pinned-mobile

### Search results - Desktop
search-desktop
### Tablet
search-tablet
### Mobile
search-mobile

### Node detail page - Desktop
details-desktop
### Tablet
details-tablet
### Mobile
details-mobile

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/content/c4m_content/c4m_content.module [hunk 1]
```diff
@@ -584,12 +584,25 @@ function c4m_content_custom_node_delete_confirm($form, &$form_state) {
  * Adding a 'Highlighted' field.
  */
 function c4m_content_field_extra_fields() {
-  $extra_fields['node']['group']['display']['highlighted'] = array(
-    'label' => t('Highlight'),
-    'description' => t('Highlight a node will toggle its promote property.'),
-    'weight' => 0,
+  $highlightable_content_types = array(
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content/c4m_content.module]: @DavidHernandez 
take this out to an external function

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content/c4m_content.module]: e.g. `c4m_message_valid_node_types()`

File: project/profiles/capacity4more/behat/features/activity_stream.feature [hunk 2]
```diff
@@ -69,7 +69,7 @@ Feature: Test activity stream
   Scenario Outline: Promote buttons should be displayed to users with access.
     Given  I am logged in as user "<user>"
     When  I visit the dashboard of group "Nobel Prize"
-    Then  I should see the ".fa-thumb-tack" element
```
Comment (ordavidil) [inline project/profiles/capacity4more/behat/features/activity_stream.feature]: You probably want just to add a line to this test and not removing this one

**Comment by DavidHernandez** [comment]
## Highlight link added to the detailed page.
### Desktop
detailed-desktop
### Tablet
detailed-tablet
### Mobile
detailed-mobile

## Highlight link on search and overview pages
### Desktop
highlight-desktop
### Tablet
highlight-tablet
### Mobile
highlight-mobile

File: project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_groups.scss [hunk 3]
```diff
@@ -862,3 +862,37 @@
...
+    display: none;
+  }
+}
+
+.group-node-actions {
+  .form-type-item {
+    margin-top: 15px;
+    margin-bottom: 5px;
+
+    .highlight-link-text {
```
Comment (ordavidil) [inline project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_groups.scss]: @DavidHernandez 
this one seems unused, please remove it ...

Comment (DavidHernandez) [inline project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_groups.scss]: @ordavidil Not unused, I missed a CSS rule there.

**Pull request merged**